src/main/java/liquibase/ext/hana/snapshot/ColumnSnapshotGeneratorHana.java

```diff
@@ -0,0 +1,39 @@
+package liquibase.ext.hana.snapshot;
+
+import liquibase.database.Database;
+import liquibase.snapshot.CachedRow;
+import liquibase.snapshot.SnapshotGenerator;
+import liquibase.snapshot.jvm.ColumnSnapshotGenerator;
+import liquibase.statement.DatabaseFunction;
+import liquibase.structure.DatabaseObject;
+import liquibase.structure.core.Column;
+
+public class ColumnSnapshotGeneratorHana extends ColumnSnapshotGenerator {
+
+    @Override
+    public int getPriority(Class<? extends DatabaseObject> objectType, Database database) {
+        return PRIORITY_DATABASE;
```

Comment:
In order to make sure that this extension priority doesn't break other databases the getPriority needs to check if the current database is hanadb,  similar to this one in bigquey: https://github.com/liquibase/liquibase-bigquery/blob/dcc3eb960fd437115d1f22bc0f95f15dba943c6d/src/main/java/liquibase/ext/bigquery/snapshot/jvm/BigQueryViewSnapshotGenerator.java#L26

Reply:
Hi @filipelautert, apologies for the delayed reply this time from my side. Good point, thanks for reviewing! I just pushed the changes to the PR/branch.